docs/README.md

```diff
@@ -119,30 +207,38 @@ When a Campaign Owner visits a Campaign page, a button named "Moderate Campaign"
 
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image20.jpg" width="auto">
 
-This button navigates to a Campaign Validation page which provides some tools that only a Campaign Owner has access to.
+This button navigates to a Campaign Moderation page which provides moderation tools that only a Campaign Owner has access to.
+
+<img src="_media/campaign-moderate.png" width="auto">
+
+### Statistics
+By clicking the *Statistics* button, the Campaign Owner can view some statistics about that specific Campaign, like total items, total annotations, annotation upvotes, contribution status  etc.
 
-<img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image21.jpg" width="auto">
+<img src="_media/campaign-moderate-statistics.png" width="auto">
 
-### View annotations by TAG
-By typing or selecting one of the available tags, like "Bass saxophone", a list of Items annotated with the chosen tag will appear.
+### Validation
+By clicking the *Validation* button the Campaign Owner can validate the campaign annotations. By typing or selecting one of the available tags, like "Bass saxophone", a list of Items annotated with the chosen tag will appear.
```

Comment:
These 2 sentences are irrelevant since we have hidden the "Campaign Bulk Validation" section (for now). Since we have only kept the "Campaign Export Criteria" section, we could change these sentences to something like:

By clicking on the "Validation" tab, the Campaign Owner can automatically validate the produced annotations, based on the feedback (upvotes/downvotes) they have received by the campaign contributors.